src/components/Layout/Layout.js

```diff
@@ -49,7 +53,23 @@ const Layout = ({ children }) => {
         <PrimaryNav />
       </div>
       <main className="l-main" id="main-content">
-        {children}
+        <div data-test="main-children">{children}</div>
+        {!cookies.release20_04 && (
+          <Notification
+            type="information"
+            close={() => {
+              setCookies("release20_04", true);
+            }}
+          >
+            This is the new JAAS Dashboard, replacing the previous Juju GUI here
+            on jaas.ai and on your local Juju Controller from Juju 2.8.{" "}
+            <span className="u-hide--small">
```

Comment:
What is this for? It doesn't appear to have any effect.

Reply:
Reduce your screen width to less than 620px wide

Reply:
Ahh I see, Do you not think those links are valuable if you're on a mobile device?

Reply:
They're less valuable than the line above.